### PR TITLE
Update README to add details for the files within the directories

### PR DESCRIPTION
**Pull request by goneall**
Signed-off-by: Gary O'Neall <gary@sourceauditor.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,11 +11,12 @@ See the file accessingLicenses.md for a description on how to programmatically a
...
+* `rdfa` - RDFa/HTML format for the license list.  The index.html file contains summary information for all licenses.  The exceptions-index.html file contains summary information for all exceptions.  The directory also contains well as single files (one per license or exception) containing data for a single file or exception named by ID.
+* `rdfnt` - RDF NT format for the license list.  The directory contains a file licenses.nt which contains all licenses and exceptions as well as single files (one per license or exception) containing data for a single file or exception named by ID.
+* `rdfturtle` - RDF turtle format for the license list.  The directory contains a file licenses.nt which contains all licenses and exceptions as well as single files (one per license or exception) containing data for a single file or exception named by ID.
+* `rdfxml` - RDF/XML format for the license list.  The directory contains a file licenses.xml which contains all licenses and exceptions as well as single files (one per license or exception) containing data for a single file or exception named by ID.
```
Comment (idimopoulos) [inline README.md]: There is a typo in the filename. The file is `licenses.rdf`.

File: README.md [hunk 2]
```diff
@@ -11,11 +11,12 @@ See the file accessingLicenses.md for a description on how to programmatically a
...
+* `jsonld` - JSON-LD format for the license list.  The directory contains a file licenses.jsonld which contains all licenses and exceptions as well as single files (one per license or exception) containing data for a single file or exception named by ID.
+* `rdfa` - RDFa/HTML format for the license list.  The index.html file contains summary information for all licenses.  The exceptions-index.html file contains summary information for all exceptions.  The directory also contains well as single files (one per license or exception) containing data for a single file or exception named by ID.
+* `rdfnt` - RDF NT format for the license list.  The directory contains a file licenses.nt which contains all licenses and exceptions as well as single files (one per license or exception) containing data for a single file or exception named by ID.
+* `rdfturtle` - RDF turtle format for the license list.  The directory contains a file licenses.nt which contains all licenses and exceptions as well as single files (one per license or exception) containing data for a single file or exception named by ID.
```
Comment (idimopoulos) [inline README.md]: There is a typo in the filename. the file is `licenses.turtle`

File: README.md (hunk 1 above)
Comment (goneall) [reply README.md]: Updated - @idimopoulos thanks for the review

File: README.md (hunk 2 above)
Comment (goneall) [reply README.md]: Corrected the typo

**Pull request merged**